example/src/app/components/freerasp-demo/freerasp-demo.component.html

```diff
@@ -0,0 +1,33 @@
+<ion-content scroll-y="true">
+  <ion-list>
+    <ion-list-header>
+      <h1>freeRASP checks:</h1>
+    </ion-list-header>
+
+    <ion-item *ngFor="let check of checks">
```

Comment:
Dajú sa tu už používať nové Angular notácie?
```suggestion
    @for (check of checks; track $index) {
    <ion-item>
    
      <ion-label [color]="check.isSecure ? 'success' : 'danger'">
        {{ check.name }}
        <p>{{ check.isSecure ? 'secure' : 'danger' }}</p>
      </ion-label>
      @if (check.name === 'Malware') {
              <malware-modal [isDisabled]="check.isSecure"> </malware-modal>
      }

      <ion-icon
        [icon]="check.isSecure ? shieldCheckmarkIcon : alertCircleIcon"
        [color]="check.isSecure ? 'success' : 'danger'"
        size="large"
      >
      </ion-icon>
    </ion-item>
    }
```